Acornfile

```diff
@@ -0,0 +1,33 @@
+services: db: image: "ghcr.io/acorn-io/mariadb:v10.11.5-1"
+
+
+containers: {
+	web: {
+		image: "docker.io/bitnami/moodle:4.3"
+		ports: publish: "8080:8080/http"
```

Comment:
There are 2 basic things missing here.
1. First of all we should definitely make this service available on secure port with self-signed certs. We shouldn't miss out on this, when it's already done in the reference [docker-compose implementation](https://github.com/bitnami/containers/blob/main/bitnami/moodle/docker-compose.yml#L21) & the dockerfile
2. Secondly, we should have at-least translated the container port 8080 to the default http port i.e. 80. So extending this

Reply:
1. Updated it to use 80 port. Also I observe that using `80:8080`  the URL the acorn platform has https enabled. I think all the certs are by default handle by acorn. When I tried using `443:8443`, after access the URL generated by acorn it gives `Internal Server Error`. So I think just using `80:8080` will work here.